src/Components/Calculator.js

```diff
@@ -0,0 +1,16 @@
+import Button from './Button';
+import Screen from './Screen';
+import './Calculator.css';
+
+const Calculator = () => {
+  const buttons = ['AC', '+/-', '%', '+', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];
+
+  return (
+    <div className="calculator">
+      <Screen className="screen" />
+      {buttons.map((button) => <Button key={button.id} text={button} />)}
+    </div>
+  );
+};
```

Comment:
- [ ] Good job on converting the `App`, `Button`, and `Screen` to a `class-based`. Kindly convert the `Calculator` component to `class-based` instead of `function-based` as requested by the previous reviewer as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).). All components should be class-based.